storybook/src/components/PasswordInput/PasswordInput.stories.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { PasswordInput } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Components/Forms/Password Input',
+  component: PasswordInput,
+  args: {
+    disabled: false,
+    invalid: false,
+    minLength: 12,
+  },
+  argTypes: {
+    disabled: {
+      description: 'Prevents interaction. Avoid if possible.',
+    },
+  },
+} satisfies Meta<typeof PasswordInput>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {}
```

Comment:
Gov.uk advises against realtime validation. Maybe wrap the input in a `form` with `novalidate`?

Reply:
Yes, added.

Reply:
Hm, this doesn't seem to be working, it still validates while you type.

Reply:
> The issue you’re encountering is likely due to the browser’s built-in CSS styling applied to the input field when the `minlength` attribute is used, regardless of the `novalidate` attribute on the form. The `novalidate` attribute disables the browser's built-in form validation, meaning it won’t prevent form submission or show default validation messages. However, it does not prevent the browser from applying its default styles based on the validation state of the input fields.
> 
> Here's what’s happening:
> 
> 1. The `minlength` attribute specifies a minimum number of characters that the user must enter. If the input value is shorter than this minimum, the input field is considered invalid.
> 2. Despite using `novalidate`, the browser still tracks the validity state of the input element and may apply default styles indicating an invalid state (such as turning the border red).

So I guess the issue here is our styling for `:invalid`, not the example itself.

Reply:
Hm, I see. Do we still want to recommend using it then? If it doesn't work the way we want it to? Both React Hook Form and Formio don't use the `minlength` attribute directly, I can imagine that's for this reason.

`minlength` is kind of similar to `maxlength` I guess, in that you can use them in form validation libraries, but not directly as attributes. Both validate too quickly, and in the case of `maxlength` it cuts off the input.

Reply:
Yes, I think I agree. It still feels like a big leap to denounce browser validation entirely, but GOV.UK makes a compelling case and we all know how bad errors before finishing typing are.

I’ve removed the attribute and overturned the guideline.

Reply:
Yeah, I think that is a good idea for now. We have to work this out in detail in our form guidelines, but that's something for later.

Without any validation attributes, we can get rid of the `<form noValidate>` decorator, I think?